src/codeu/chat/server/Server.java

```diff
@@ -129,10 +129,39 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
         Serializers.INTEGER.write(out, NetworkCode.NEW_USER_RESPONSE);
         Serializers.nullable(User.SERIALIZER).write(out, user);
 
-
       }
     });
 
+    // Add Member - A creator or owner wants to add a new member to a conversation.
+    this.commands.put(NetworkCode.ADD_MEMBER_REQUEST,  new Command() {
+      @Override
+      public void onMessage(InputStream in, OutputStream out) throws IOException {
+
+        final String user = Serializers.STRING.read(in);
+        final Uuid conversation = Uuid.SERIALIZER.read(in);
+        //psuedo code don't know how to implement?
+        controller.addMember(user, conversation);
+
+        Serializers.INTEGER.write(out, NetworkCode.ADD_USER_RESPONSE);
+
```

Comment:
Just clean up some of the spacing to preserve the minimal amount of lines of code as possible.